homeassistant/components/wake_on_lan/__init__.py

```diff
@@ -24,9 +35,41 @@
     }
 )
 
+SWITCH_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_MAC): cv.string,
+        vol.Optional(CONF_BROADCAST_ADDRESS): cv.string,
+        vol.Optional(CONF_BROADCAST_PORT): cv.port,
+        vol.Optional(CONF_HOST): cv.string,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_OFF_ACTION): cv.SCRIPT_SCHEMA,
+    }
+)
+
+
+CONFIG_SCHEMA = vol.Schema(
+    {vol.Optional(DOMAIN): vol.All(cv.ensure_list, [vol.Schema(SWITCH_SCHEMA)])},
+    extra=vol.ALLOW_EXTRA,
+)
```

Comment:
This is violating ADR-0010

Reply:
Reverted in #82184

Reply:
The errors looked unrelated to this. But maybe i missed something.
Sorry should have realized it was in violation of ADR-0010.